go_sequencer/database/statedb/merkel_tree.go

```diff
@@ -24,8 +25,10 @@ func (s *StateDB) GetMTRoot(treeType enum) *big.Int {
 	var root *big.Int
 	if treeType == Account {
 		root = s.AccountTree.Root().BigInt()
```

Comment:
Just an opinion, Instead of having nested if statements here in `GetMTRoot`, We can add individual specific functions in the respective account, vouch, score.go file.

eg - GetMTRootVouch() It'll make the code clean and also eliminate the need to maintaining an extra treeType enum.

Let me know your thoughts.

Thanks